refarch-integrations/refarch-dms-integration/refarch-dms-integration-core/src/main/java/de/muenchen/refarch/integration/dms/adapter/out/s3/S3Adapter.java

```diff
@@ -107,13 +107,16 @@ private long getFileSize(final String filePath) throws DocumentStorageException
         }
     }
 
+    @SuppressWarnings("PMD.ExceptionAsFlowControl")
```

Comment:
I think instead of suppression here we could refactor this method. I agree with PMD that we shouldn't use exception throwing solely for control flow.

Reply:
That's more coming because of the abstraction and Server and ClientException only occur for the rest adapter and not the java one. Probably we should talk about that